packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -151,12 +154,12 @@ export const CoinjoinStatus = ({
     const togglePause = useCallback(async () => {
         if (isPaused) {
             setIsLoading(true);
-            await restoreSession();
+            await dispatch(restoreCoinjoinSession(account!));
```

Comment:
how do you know, that `account` is always defined here? What about some `if(!account) return null` check at the top?